src/query-builder/InsertQueryBuilder.ts

```diff
@@ -226,6 +226,29 @@ export class InsertQueryBuilder<Entity> extends QueryBuilder<Entity> {
         return this;
     }
 
+    /**
+     * Adds additional ignore statement supported in databases.
```

Comment:
Should we add information which database engines suport such statements? Like in line 222.